### PR TITLE
Expanded Northstar Robotics Area into Maints (Inclusion of Mech Bay)

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Changes the Northstar robotics area, some of maintenence, and a tiny bit of Circuits to expand the robotics lab to fit proper equipment. 

The previous 'workshop area' is now the Mech Bay. It is properly equipped with two mech chargers, two exosuit fabricators, a welding fuel tank, and no tables under the shutters to allow exosuits to directly leave.

The usual robotics area now is located just north of the mech bay, and has more toolboxes, tools, and robot parts. It also now contains the robo-drobe, which was previously missing.  It also now has privacy shutters.

This expansion has slightly changed the layout. The lab replaces a small maints area with only a few random crates, while retaining maints access from robotics. Additionally, a small portion of Circuits has been bumped up to make both workplaces more streamlined.

TLDR, Robotics is much better with more stuff at the cost of a maints room and a bit of circuits

## Why It's Good For The Game

The Northstar robotics lab we have currently is objectively underequipped and intolerable. It has only one forge, lacks a mech charger, a drobe, a sniffer, basic bot parts, and protective welding gear, is super cramped, the shutters are blocked by reinforced tables, and no privacy shutters. This expansion adds all of these important aspects into the department, to make it equally as fun as the other stations currently in rotation.

## Changelog

add: Expanded Northstar robotics area
/:cl:

### CODE REVIEW

**Comment by EEASAS** [comment]
Fix your changelog, it needs to have an "add:" before the changelog. like this "add: expanded robotics"

**Comment by VelocityMagnet** [comment]
Small problem with it. For some reason, whenever i load the modified map, very specific items' sprites are just errors. Screwdriver, toolbelt, and after a bit of exploring, benches also have error sprites. I have no clue how this happened, can anyone provide some insight?

**Comment by MrMelbert** [comment]
> Small problem with it. For some reason, whenever i load the modified map, very specific items' sprites are just errors. Screwdriver, toolbelt, and after a bit of exploring, benches also have error sprites. I have no clue how this happened, can anyone provide some insight?

Hm, are you on latest byond beta (515.1632)? If so you should downgrade

You also seem to have some error in the map so that might be contributing

image

**Comment by Blacklist897** [comment]
that room you expanded is not robotics
look right of medical

**Comment by Cheshify** [comment]
Hey there, North Star map person here. That room actually isn't robotics as Blacklist has pointed out. That's just a small workshop for Science to use, and for robotics to quantum pad to.

**Comment by VelocityMagnet** [comment]
After seeing the other robotics area, I agree it is sufficient for roboticists and needs no expanded workshop. However, it's location is a bit disconnected. Pull closed.
